<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pokemon</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon_pokemon96.png</normaloff>icon/icon_pokemon96.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_mypokemon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_oppokemon">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_hp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(255, 0, 0);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>HP %p / 100</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_mp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(0, 85, 255);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>MP %p / 100</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_op_mp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(0, 85, 255);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>MP %p / 100</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_op_hp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(255, 0, 0);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>HP %p / 100</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>370</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(53, 100, 173);
font: 14pt &quot;휴먼매직체&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>370</y>
      <width>130</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(53, 100, 173);
	color: rgb(255, 203, 5);
	border-color: rgb(53, 100, 173);
	border-radius: 10px;
	font: 24pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	color: rgb(255, 255, 127);
	background-color: rgb(0, 0, 255);
}
</string>
    </property>
    <property name="text">
     <string>전투 종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_skill2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	color: rgb(0, 0, 255);
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_skill3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	color: rgb(0, 0, 255);
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_skill4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);

	color: rgb(53, 100, 173);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	color: rgb(0, 0, 255);
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_skill5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	color: rgb(0, 0, 255);
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mp">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_skill1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: rgb(53, 100, 173);
	border-radius: 10px;
	border-width: 5px;
	border-color: red;
	font: 12pt &quot;휴먼매직체&quot;;
}
QPushButton:hover{
	color: rgb(0, 0, 255);
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_mydamaged">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_opdamaged">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(53, 100, 173);
font: 24pt &quot;휴먼매직체&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_my">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	font: 10pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_op">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	font: 10pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>listWidget</zorder>
   <zorder>label_mypokemon</zorder>
   <zorder>label_oppokemon</zorder>
   <zorder>progressBar_hp</zorder>
   <zorder>progressBar_mp</zorder>
   <zorder>progressBar_op_mp</zorder>
   <zorder>progressBar_op_hp</zorder>
   <zorder>pushButton_back</zorder>
   <zorder>pushButton_skill2</zorder>
   <zorder>pushButton_skill3</zorder>
   <zorder>pushButton_skill4</zorder>
   <zorder>pushButton_skill5</zorder>
   <zorder>pushButton_hp</zorder>
   <zorder>pushButton_mp</zorder>
   <zorder>pushButton_skill1</zorder>
   <zorder>label_mydamaged</zorder>
   <zorder>label_my</zorder>
   <zorder>label_op</zorder>
   <zorder>label_opdamaged</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
